.prettierignore

```diff
@@ -9,5 +9,8 @@ public
 /packages/xsnap/moddable
 /packages/xsnap/xsnap-native
 
+# copied from upstream
+/packages/xsnap/lib/object-inspect.js
```

Comment:
@michaelfig why is this copied from upstream instead of sourced from the `object-inspect` dependency in package.json?

Reply:
I could have commented that better.  It's not a verbatim copy, it's the `object-inspect.js` with an unnecessary CommonJS `require('./util.inspect')` removed since it doesn't work under XS.

Reply:
Cool. What's the reason to prevent prettying it? If it's to make it easier to update from upstream, WDYT of using `patch` and to make it work under XS?

Reply:
> Cool. What's the reason to prevent prettying it? If it's to make it easier to update from upstream, WDYT of using `patch` and to make it work under XS?

That's fine with me, if you're bored and want something to do. 😄   Tests cover it, so it's a safe thing to try.

Reply:
`patch-package` can't be used, though, because people might get this code directly via NPM's `@agoric/xsnap`.